judgels-frontends/raphael/src/components/MathjaxWrapper/MathjaxWrapper.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+declare global {
+  interface Window {
+    MathJax: any;
+  }
+}
+
+export class MathjaxWrapper extends React.Component {
+  componentDidMount() {
+    this.insertScript('MathJax-config', '/var/conf/mathjax-config.js');
+    this.insertScript('MathJax-script', '/mathjax/tex-svg.js');
```

Comment:
Does this work in development mode?

Reply:
Yes, this work, the mathjax folder is inside the public folder.